app/Providers/AuthServiceProvider.php

```diff
@@ -6,6 +6,7 @@
 use Illuminate\Foundation\Support\Providers\AuthServiceProvider as ServiceProvider;
 
 class AuthServiceProvider extends ServiceProvider
+
 {
```

Comment:
Opening brace of a class must be on the line following the class declaration; found 1 line(s)